tasks/zadanie_24.php

```diff
@@ -44,3 +44,31 @@
 $products = $params[0]; // tej linijki nie ruszamy :)
 $filterPrice = $params[1]; // tej linijki nie ruszamy :)
 $filterMode = $params[2]; // tej linijki nie ruszamy :)
+
+
+
+$filterProducts = array_filter($products, function($product) use ($filterPrice, $filterMode) {
+
+    if ($filterMode === 'gte') {
+        return $product['price'] >= $filterPrice;
+    }
+
+    if ($filterMode === 'lte') {
+        return $product['price'] <= $filterPrice;
+    }
+
+    return true;
+});
+
+if ($filterMode === 'gte') {
+    usort($filterProducts, fn($a, $b) => $a['price'] <=> $b['price']);
+} elseif ($filterMode === 'lte') {
+    usort($filterProducts, fn($a, $b) => $b['price'] <=> $a['price']);
+}
+
+foreach ($filterProducts as $product) {
+    $name = $product['name'];
+    $price = $product['price'];
+
+    echo $name . ': ' . number_format($price / 100, 2, ',', ' ') . " zł" . PHP_EOL;
```

Comment:
Tutaj dla funkcji number_format należy usunąć tą spację jako rozdzielnik tysięcy ;)
```suggestion
    echo $name . ': ' . number_format($price / 100, 2, ',', '') . " zł" . PHP_EOL;
```
**Oczekiwany wynik:**
Laptop: 5000,00 zł
Klawiatura: 400,00 zł
Monitor: 4000,00 zł

**Uzyskany wynik:**
Klawiatura: 400,00 zł
Monitor: 4 000,00 zł
Laptop: 5 000,00 zł